server/sonar-server/src/test/java/org/sonar/server/usergroups/ws/RemoveUserActionTest.java

```diff
@@ -276,4 +310,13 @@ private boolean isUserInGroup(UserDto userDto, GroupDto groupDto) {
       .anyMatch(dto -> dto.getLogin().equals(userDto.getLogin()));
   }
 
+  private UserDto insertAnAdministratorInDefaultOrganization() {
+    return insertAnAdministrator(db.getDefaultOrganization());
+  }
+
+  private UserDto insertAnAdministrator(OrganizationDto org) {
```

Comment:
you could add method `insertAdminByUserPermission` to `UserDbTester` in the same flavour there is `insertRootByUserPermission`